pages/sdk-protocol-kit/guides/safe-signatures.md

```diff
@@ -0,0 +1,848 @@
+| **safeAddress3_4** | 0xb3b3862D8e38a1E965eb350B09f2167B2371D652 |
+| **owner1**         | 0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1 |
+| **owner2**         | 0xFFcf8FDEE72ac11b5c542428B35EEF5769C409f0 |
+| **signerSafe1_1**  | 0x215033cdE0619D60B7352348F4598316Cc39bC6E |
+| **owner3**         | 0x22d491Bde2303f2f43325b2108D26f1eAbA1e32b |
+| **signerSafe2_3**  | 0xf75D61D6C27a7CC5788E633c1FC130f0F4a62D33 |
+| **owner4**         | 0xE11BA2b4D45Eaed5996Cd0823791E0C93114882d |
+| **owner5**         | 0xd03ea8624C8C5987235048901fB614fDcA89b117 |
+
+## Transactions
+
+### Creating the transaction object
+
+We can sign transactions using the `protocol-kit` by creating an instance of the `Safe` class. Here's how we can do it:
+
+```typescript
+import Safe from '@safe-global/protocol-kit';
+
+// You can use any compatible adapter, such as Web3Adapter or EthersAdapter
+const protocolKit = await Safe.create({
+  ethAdapter: ethAdapter1,
+  safeAddress: safeAddress3_4,
+});
+```
+
+The `ethAdapter1` is bound to `owner1`. In the examples provided in this article, we will have 5 adapters, each one bound to an owner.
+
+| Adapter     | Bound to |
+| ----------- | -------- |
+| ethAdapter1 | owner1   |
+| ethAdapter2 | owner2   |
+| ethAdapter3 | owner3   |
+| ethAdapter4 | owner4   |
+| ethAdapter5 | owner5   |
+
+After obtaining the `protocolKit` instance, we can use `createTransaction()` to generate a transaction object.
+
+```typescript
+// Create the transaction. Send 0.01 ETH
+const safeTransactionData: SafeTransactionDataPartial = {
+  to: '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1',
+  value: '100000000000000000', // 0.01 ETH
+  data: '0x',
+};
+
+let safeTx = await protocolKit.createTransaction({
+  transactions: [safeTransactionData],
+});
+```
+
+The `safeTx` is an `EthSafeTransaction` object that stores the transaction data (`data` prop) and a map of owner-signature pairs (`signatures` prop).
+
+```typescript
+class EthSafeTransaction implements SafeTransaction {
+  data: SafeTransactionData
+  signatures: Map<string, SafeSignature> = new Map()
+...
+// Other props and methods
+}
+```
+
+### Generating transaction signatures
+
+Now that we've the transaction object (`safeTx`), it's time to add the necessary signatures.
+
+#### Creating an ECDSA signature
+
+---
+
+We will sign with `owner1` and `owner2` using the `signTransaction()` method. This method add new signatures to the `signatures` map by using the transaction `data` as input.
+
+It's possible to use several signing methods, such as `ETH_SIGN` (eth_sign), `ETH_SIGN_TYPED_DATA_V4` (eth_signTypedData_v4), ...etc. The default signing method is `ETH_SIGN_TYPED_DATA_V4`.
+
+`ETH_SIGN` produces a regular [EIP-191](https://eips.ethereum.org/EIPS/eip-191) signature. `ETH_SIGN_TYPED_DATA_V4` produces a signature that's compatible with the [EIP-712](https://eips.ethereum.org/EIPS/eip-712) standard. The EIP-712 standard is used by Safe contracts to verify the signatures.
```

Comment:
EIP-191 and EIP-712 are about something other than the signatures. They define how the message for signing is generated.